src/python/test/test_dxclient.py

```diff
@@ -2728,6 +2728,40 @@ def main():
         shell.close()
         self.assertEqual(3, shell.exitstatus)
 
+    @unittest.skipUnless(testutil.TEST_RUN_JOBS, 'skipping tests that would run jobs')
```

Comment:
Can you add as well unit tests of dx-unpack that run locally? I imagine that will help you to track down a lot more of these quoting issues locally. In addition to taking possibly minutes (instead of milliseconds), this test has the disadvantage that you won't see it pass until the fixed dx-unpack has actually been redeployed to staging.